### ISSUE TITLE
Login Problems

### ISSUE THREAD

**Issue by asmaasehli** [ageuk/companion-app#133]
I sometimes have issues where I can't login, and have to clear Local Storage in order to.

See screenshot below.

![authorisation missing](https://cloud.githubusercontent.com/assets/20186112/17699972/bc4e7d70-63bb-11e6-80a9-a6f06d076c74.png)

**Comment by andrewMacmurray** [comment]
thanks @asmaasehli, what happened before this? Did you do anything like clear website data or clear cookies? The reason that message would appear was if the cookie was missing so trying to figure out why it can't find it

**Comment by asmaasehli** [comment]
I didn't do anything before this - the last time I looked at the app was when you were in, then I went back onto it today and it happened so I cleared Local Storage as you've shown me how to, and it is now working fine - didn't touch the cookies (because I don't know how!)

@andrewMacmurray

**Comment by andrewMacmurray** [comment]
@asmaasehli ah ok that's interesting, will have a think about what could have caused that. For now if you clear your browsing data (it's in chrome preferences - click on `settings` then at the bottom `show advanced settings` and click on `clear browsing data`

**Comment by andrewMacmurray** [comment]
![screen-shot-2016-08-16-at-14 26 28](https://cloud.githubusercontent.com/assets/14013616/17700423/8c964fac-63bd-11e6-9e95-e3b9bdd77928.jpg)

**Comment by asmaasehli** [comment]
@andrewMacmurray - okay great thanks.

**Comment by asmaasehli** [comment]
Whoops.

**Comment by andrewMacmurray** [comment]
@asmaasehli 😆  we've all done that before!

**Comment by andrewMacmurray** [comment]
Hey @asmaasehli, the fix we made has been merged in, that should fix the problems you were having.

Just to be sure, clear your browsing data (instructions are in a few comments ⬆️  ) and try it again.

If you do find this is still a problem later on, reopen the issue and `@` the new people.

cheers, hope the testing phase goes well! 😄

**Comment by andrewMacmurray** [comment]
I hear there are still problems with the login?

Given conversation I've been reading via email:

> Because the app (our side) is served under an .ageuk.org.uk sub-domain there seems to be conflicts with other things that have set cookies across the whole domain (including all sub-domains). One example for me is Maxymiser and my login auth clashing with steps’. Another is ClickTale on the Age UK website - if they have accessed the Age UK website this may cause it. When trying to sync you get a 401 Unauthorised. I presume there must be a way round this.

It may be worth changing the name of the cookie from `ageuk_token` to something else

here: https://github.com/ageuk/companion-app/blob/master/public/src/js/helpers.js#L436
and here: https://github.com/ageuk/companion-app/blob/master/lib/hapiLoginConfig.js#L32

**Comment by iteles** [comment]
Agreed, we'll try this out as a solution, @Danwhy has a little time allocated to it on Monday 👍

**Comment by iteles** [comment]
@rikardoH Katherine tried changing the cookie name so you'll need to re-deploy to AWS so the change is reflected please 😊
